### PR TITLE
{phys}[GCCcore/13.2.0] Lime 1.3.2

### PR DESCRIPTION
**Pull request by jkuhl-uni**
Hey,
this time, I am, in fact, contributing something for my own research community :D
This is a very simple library. For the LIME packaging scheme.

### CODE REVIEW

**Review by jfgrimm — CHANGES_REQUESTED** [review]
minor nitpicks

File: easybuild/easyconfigs/l/LIME/LIME-1.3.2-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        "include/lime_writer.h",
+    ],
+    'dirs': [
+        "bin",
+        "include",
+        "lib",
+        "lib64"
+    ],
+}
+
```
[the remark this answers is not in the document]
Comment (jkuhl-uni) [reply easybuild/easyconfigs/l/LIME/LIME-1.3.2-GCCcore-12.3.0.eb:52]: In principle, one could execute one of the bins, the thing is, though, that there is no `--help` or similar, sorry about that. I thought about it, but the alternative is, I guess to download a test file and do something like `lime_list_contents <file> && lime_extract_record ...`

[the remark this answers is not in the document]
Comment (jfgrimm) [reply easybuild/easyconfigs/l/LIME/LIME-1.3.2-GCCcore-12.3.0.eb:52]: that's fine, don't worry

**Comment by jfgrimm** [comment]
@boegelbot: please test @ generoso

**Comment by jfgrimm** [comment]
@boegelbot: please test @ jsc-zen3

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login2.viking2.yor.alces.network - Linux Rocky Linux 8.9, x86_64, AMD EPYC 7643 48-Core Processor, Python 3.6.8
See https://gist.github.com/jfgrimm/363b4c181823a86fe26af4879dea7bd7 for a full test report.

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
Going in, thanks @jkuhl-uni!

**Pull request merged**